frontend/packages/container-security/src/plugin.tsx

```diff
@@ -1,55 +0,0 @@
-import {
-  Plugin,
-  ModelDefinition,
-  ModelFeatureFlag,
-  ResourceListPage,
-  ResourceDetailsPage,
-} from '@console/plugin-sdk';
-import { ContainerSecurityFlag } from './const';
-import { ImageManifestVulnModel } from './models';
-
-type ConsumedExtensions =
-  | ModelDefinition
-  | ModelFeatureFlag
-  | ResourceListPage
-  | ResourceDetailsPage;
-
-const plugin: Plugin<ConsumedExtensions> = [
-  {
-    type: 'ModelDefinition',
```

Comment:
Note this extension was removed without replacement. I could not issues that appeared after removal but this may require more testing